torch/_torch_docs.py

```diff
@@ -5670,6 +5670,29 @@ def merge_dicts(*dicts):
     tensor([ 1., -1.,  0.,  1.])
 """.format(**common_args))
 
+add_docstr(torch.signbit,
+           r"""
+signbit(input, *, out=None) -> Tensor
+
+Tests if each element of :attr:`input` has its sign bit set (is less than zero) or not.
+
+.. math::
+    \text{out}_{i} = \operatorname{sgn}(\text{input}_{i})
```

Comment:
\operatorname{signbit} ?

Reply:
The formulas has been deleted, Sorry for my carelessness.